src/libs/OptionsListUtils/index.ts

```diff
@@ -643,7 +643,8 @@ function getLastMessageTextForReport({
     } else if (isTaskAction(lastReportAction)) {
         lastMessageTextFromReport = formatReportLastMessageText(getTaskReportActionMessage(lastReportAction).text);
     } else if (isCreatedTaskReportAction(lastReportAction)) {
-        lastMessageTextFromReport = getTaskCreatedMessage(lastReportAction);
+        const lastReportActionChildReport = lastReportAction?.childReportID ? allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${lastReportAction.childReportID}`] : undefined;
+        lastMessageTextFromReport = getTaskCreatedMessage(lastReportAction, lastReportActionChildReport);
```

Comment:
should use existing `getReportOrDraftReport` util function instead (already imported in this component)

```suggestion
        lastMessageTextFromReport = getTaskCreatedMessage(lastReportAction, getReportOrDraftReport(lastReportAction?.childReportID));
```